### PR TITLE
framework/st_things: Remove warning message at build message.

### PR DESCRIPTION
**Pull request by harish-191**
1. Remove warning message from St things framework which is generated
   at build time.
2. Remove warning message from Iotivity which is generated at build
   time.

Signed-off-by: harish191 <harish.191@samsung.com>

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [a8630794fffe9ddbe2c674f17767d47a9b6ae06c] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [a8630794fffe9ddbe2c674f17767d47a9b6ae06c] - Code Rule Check OK.

File: framework/src/st_things/st_things.c [hunk 1]
```diff
@@ -403,7 +403,7 @@ int st_things_push_notification_to_cloud(const char *target_uri, st_things_repre
 	} else {
 		OCRepPayloadSetUri(resp_rep->payload, target_uri);
 		OCRepPayloadSetPropInt(resp_rep->payload, SEC_ATRRIBUTE_PUSH_MESG_MSGID, 0);
-		OCRepPayloadSetPropString(resp_rep->payload, SEC_ATRRIBUTE_PUSH_MESG_PROVIDERID, OCGetServerInstanceIDString());
+		OCRepPayloadSetPropString(resp_rep->payload, SEC_ATRRIBUTE_PUSH_MESG_PROVIDERID, (char *)OCGetServerInstanceIDString());
```
Comment (SenthilKumarGS) [inline framework/src/st_things/st_things.c]: OCGetServerInstanceIDString() returns 'const char *' and OCRepPayloadSetPropString() expects 'const char *'.
So typecast doesn't seem to be required.
Could you please re-check?

**Comment by sunghan-chang** [comment]
@harish-191 I found many `Handle review comment` commit from you. You MUST NOT make it to resolve review comment. It is usless to another developer. You must merge it into original commit.
And I found some violations of our commit rules as well.
Please find our commit rules first.

**Comment by seinfra** [comment]
Target : [963203bb99889a728ffc3ebd4c4337c17079ad36] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [963203bb99889a728ffc3ebd4c4337c17079ad36] - Code Rule Check Result:
framework/src/st_things/things_stack/cloud/cloud_manager.c:2142: ERROR: [SPC_M_OPR] space prohibited before that ':' (ctx:WxE)
framework/src/st_things/things_stack/easy-setup/easysetup.c:162: ERROR: [SPC_M_OPR] space prohibited before that ':' (ctx:WxE)
framework/src/st_things/things_stack/easy-setup/resource_handler.c:1574: ERROR: [SPC_M_OPR] space prohibited before that ':' (ctx:WxE)
framework/src/st_things/things_stack/framework/things_security_manager.c:513: ERROR: [IDT_M_TAB] code indent should use tabs where possible
framework/src/st_things/things_stack/framework/things_security_manager.c:513: ERROR: [IDT_M_TAB] please, no spaces at the start of a line

**Comment by seinfra** [comment]
Target : [46bf51c699030df043772a79bccaf10ae9f69986] - Code Rule Check Result:
framework/src/st_things/things_stack/cloud/cloud_manager.c:2123: ERROR: [SPC_M_OPR] space prohibited before that ':' (ctx:WxE)
framework/src/st_things/things_stack/easy-setup/resource_handler.c:1543: ERROR: [SPC_M_OPR] space prohibited before that ':' (ctx:WxE)
framework/src/st_things/things_stack/framework/things_security_manager.c:512: ERROR: [IDT_M_TAB] code indent should use tabs where possible
framework/src/st_things/things_stack/framework/things_security_manager.c:512: ERROR: [IDT_M_TAB] please, no spaces at the start of a line

**Pull request closed without merging**